lib/utils.pm

```diff
@@ -2927,4 +2928,25 @@ sub write_sut_file {
     assert_script_run("curl -v -o $path $url");
 }
 
+=head2 handle_gnome_memory_ge_4g
+
+  handle_gnome_memory_ge_4g([fatal_flag => $fatal_flag])
+
+Handle gnome test on sle, system's ram must be at least 4GB
+https://progress.opensuse.org/issues/153808.
+
+=cut
+
+sub handle_gnome_memory_ge_4g {
+    my (%args) = @_;
+    $args{fatal_flag} //= 1;
+
+    my $os_handled = is_tumbleweed || is_leap('>=15.5') || is_sle('>=15-SP5');
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
Thanks Oliver, you are correct, let me fix it